server.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const cors = require('cors');
+const helmet = require('helmet');
+const router = require('./data/router.js');
+
+const server = express();
+
+server.use(express.json());
+server.use(cors());
+server.use(helmet());
```

Comment:
security first--run helmet right after you init your server so it has time to set it's Header setting before anything else is run